tests/foreman/destructive/test_capsule_loadbalancer.py

```diff
@@ -142,7 +142,7 @@ def loadbalancer_setup(
 def test_loadbalancer_register_client_using_ak_to_ha_proxy(loadbalancer_setup, rhel7_contenthost):
     """Register the client using ak to the haproxy
 
-    :id: bd3c2e50-18e2-4be7-8a7f-c32472e17c61
+    :id: 2125f9f6-62a4-4d84-910c-57737a617cd7
```

Comment:
This is updated as part of pre-commit check.

Reply:
can you check if there's any test with same uuid and revert this? non-blocking